message.py

```diff
@@ -0,0 +1,30 @@
+class Message:
+    """ A Message Class that contain a message and is details """
+
+    def __init__(self, msg_id: int, sender: str, msg_body=''):
```

Comment:
This __init__ is redundant, read about dataclass decorator and use it - 
https://docs.python.org/3/library/dataclasses.html